src/thread.c

```diff
@@ -103,8 +103,8 @@ static int luv_thread_arg_set(lua_State* L, luv_thread_arg_t* args, int idx, int
       }
       break;
     default:
-      fprintf(stderr, "Error: thread arg not support type '%s' at %d",
-        lua_typename(L, arg->type), i);
+      fprintf(stderr, "Error: thread arg #%d not support %s type\n",
```

Comment:
can we avoid `stderr` here as well? IIUC, `lua_error` would be ok here as there is a caller to return to, so it won't bring down the entire thread?

Reply:
ok, I need more time to think and test, this api must support called safe in uv threadpools.